### PR TITLE
Add merge path to schedule mi workflows

### PR DESCRIPTION
**Pull request by xtuchyna**
## Related Issues and Dependencies
Related to https://github.com/thoth-station/mi-scheduler/pull/190

## This introduces a breaking change

- [ ] Yes
- [X] No

## This Pull Request implements
Add new merge path variable to the scheduling of mi workflows

### CODE REVIEW

**Comment by xtuchyna** [comment]
Also related to https://github.com/thoth-station/mi/pull/439

File: thoth/common/openshift.py [hunk 1]
```diff
@@ -1472,6 +1472,7 @@ def schedule_mi_workflow(
         knowledge_path: Optional[str] = None,
         mi_used_for_thoth: Optional[bool] = False,
         mi_merge: Optional[bool] = False,
+        mi_merge_path: Optional[str] = None,
```
Comment (pacospace) [inline thoth/common/openshift.py:1475]: it can be None?

Comment (xtuchyna) [reply thoth/common/openshift.py:1475]: yes, if `mi_merge = True`  then the default location will be used.
If it is `False`, then we don't care and it is None.

File: thoth/common/openshift.py [hunk 2]
```diff
@@ -1489,6 +1490,7 @@ def schedule_mi_workflow(
             "KNOWLEDGE_PATH": knowledge_path,
             "MI_THOTH": "1" if mi_used_for_thoth else "0",
             "MI_MERGE": "1" if mi_merge else "0",
+            "MI_MERGE_PATH": mi_merge_path,
```
Comment (fridex) [inline thoth/common/openshift.py:1493]:
```suggestion
            "MI_MERGE_PATH": mi_merge_path or "",
```

I think we assign an empty string if a parameter is `None`. Is that OK here (I don't have mi context here)?

Comment (xtuchyna) [reply thoth/common/openshift.py:1493]: Yes, it is okay

File: thoth/common/openshift.py (hunk 1 above)
Comment (xtuchyna) [reply thoth/common/openshift.py]: @pacospace wdyt?

**Review by fridex — APPROVED** [review]
lgtm, thanks! 👍🏻

**Pull request merged**